PyRoute/StatCalculation.py

```diff
@@ -208,6 +217,10 @@ def calculate_statistics(self, ally_match):
             for uwpStats in uwpName.values():
                 self.per_capita(None, uwpStats)
 
+        self.galaxy.trade.is_sector_trade_balanced()
+        self.galaxy.trade.is_sector_pass_balanced()
```

Comment:
This is were we should call the `is_allegiance_trade_balanced()` methods?

Reply:
Yes-  lemme bolt them in